doc/mosfire.rst

```diff
@@ -116,9 +116,11 @@ flag **bound_detector** is set to **True**.
 
 Flat Fielding
 -------------
-Although ``PypeIt`` is able to recognize flat frames taken with the lamps off (and those frame will be
-assigned the frame type ``pixelflat``, ``trace``, but not ``illumflat``), currently the lamps off flats
-are not used to correct the dome flats for the thermal emission generated by the dome during K-band observations.
+``Pypeit`` is able to recognize flat frames taken with the lamps off, and it will assign them the
+``thermalflat`` frame type. Whenever ``thermalflat`` frames are identified in the PypeIt file, ``Pypeit``
+will subtract them from the Master Pixel Flat and Master Illumination Flat. It is responsibility of the
```

Comment:
This cannot be what we are doing. We don't want to subtract the Masters. We need to subtract before the masters are constructed. Otherwise, this entirely defeats the purpose.

Reply:
Fixed